cloudevents/extensions/expirytime.md

```diff
@@ -0,0 +1,58 @@
+# Expiry Time Extension
+
+This extension provides a mechanism to hint to [consumers](../spec.md#consumer)
+or [intermediaries](../spec.md#intermediary) a time after which an
+[event](../spec.md#event) can be ignored.
+
+In distributed systems with message delivery guarantees, events might be delivered
+to a consumer some significant amount of time after an event has been sent.
+In this situation, it might be desirable to ignore events that
+are no longer relevant. The [`time` attribute](../spec.md#time) could be used
+to handle this on the consumer side but can be tricky if the logic varies
+depending on the event type or producer.
+
+## Attributes
+
+### expirytime
+
+- Type: `Timestamp`
+- Description: Timestamp indicating an event is no longer useful after the
+  indicated time.
+- Constraints:
+  - REQUIRED
+  - If present, MUST adhere to the format specified in
```

Comment:
I don't think this bullet point is required. Firstly, it's not necessarily being represented as a string anyway, and secondly this is really just part of the `Timestamp` type definition. (In other words, I'd let the type part dictate the rest.)